foreshadow/intents/numeric.py

```diff
@@ -55,27 +53,26 @@ def transform(self, X, y=None):
 
     @classmethod
     def column_summary(cls, df):  # noqa
-        data = df.iloc[:, 0]
-        nan_num = int(data.isnull().sum())
+        result = standard_col_summary(df)
 
         data_transformed = pd.to_numeric(df.iloc[:, 0], errors="coerce")
-        invalid_num = int(data_transformed.isnull().sum() - nan_num)
-        outliers = get_outliers(data_transformed).values.tolist()
-        mode, top10 = mode_freq(data)
+        invalid_pct = (
+            data_transformed.isnull().sum() * 100.0 / result["count"]
+            - result["nan_pct"]
+        )
```

Comment:
What does `pct` stand for? Can we rename it more explicitly?

Reply:
'pct' represents percent here. I will change it to be more explicit.